tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -79,6 +79,10 @@ Your service endpoint region is selected by default. Have a quick check and clic
 
 ### Step 3. Create an AWS interface endpoint
 
+> **Note:**
+>
+> The endpoint service of each dedicated cluster will be automatically created for newly created after March 28.
```

Comment:
Adding year here to make it more precise

Reply:
done